ldm/dream/server.py

```diff
@@ -102,9 +103,20 @@ def image_done(image, seed, upscaled=False):
                         {'event':action,'processed_file_cnt':f'{x}/{iterations}'}
```

Comment:
Probably should be `processed_file_count`, also since this is intended to be consumed by JS, maybe this should be `processedFileCount`.

Reply:
That's not related to this PR.